examples/where/src/lemmas.k.md

```diff
@@ -1,5 +1,8 @@
 ```k
 rule A -Word B <=Int A => #rangeUInt(256, A -Int B)
   requires #rangeUInt(256, A) andBool #rangeUInt(256, B)
+
+rule A <=Int chop(A +Int B) => #rangeUInt(256, A +Int B)
+  requires #rangeUInt(256, A) andBool #rangeUInt(256, B)
```

Comment:
Is the `requires` really required? Seems like it might be redundant.

Reply:
Also, when does this rule help?

Reply:
The rule is needed to help `K` deal with the overflow checking introduced by solidity. The requires may not be needed, I have not tried without them. I'll try and run the spec without them and see what happens.

Reply:
turns out they were not required. removed. thx!